src/components/App/index.tsx

```diff
@@ -240,7 +224,8 @@ export default class App extends Component<Props, State> {
     }
   }
 
-  async updateImage(index: number): Promise<void> {
+  async updateImage(index: number, options: UpdateImageOptions = {}): Promise<void> {
+    const { skipPreprocessing = false } = options;
```

Comment:
Syntactic nit: Not a fan of this visually. Nbd, especially since it’s only one option, but I prefer

```js
options = {...defaultUpdateImageOptions, ...options};
```

WDYT?

Reply:
Doesn't that just introduce more scrolling to find out what the defaults are?

Reply:
(fwiw our lint rules don't allow argument overwriting)

Reply:
I feel like we had this conversation before. It probably does, but I find this incredibly noise, especially with more options. As I said, nbd.

Reply:
Last time we had this discussion we settled on the pattern above 😄. But yeah, it was a compromise. I initially put the destructuring in the args, which is kinda ugly, and until it pops up as part of autocomplete, I'm not bothered.